pulpcore/tests/functional/api/test_status.py

```diff
@@ -44,6 +44,10 @@
             },
         },
     },
+    "content_settings": {
+        "content_origin": "http://localhost:5001",
+        "content_path_prefix": "/pulp/content/",
+    },
```

Comment:
This is not correct. It should contain JSON schema types, not constants.

Reply:
Something like this:

```suggestion
    "content_settings": {
        "type": "object",
        "properties": {
            "content_origin":  {"type": "string"},
            "content_path_prefix": {"type": "string"},
        }
    },
```

Reply:
Ha, perfect! And this is why we do reviews :)